tests/atest/transformers/GenerateDocumentation/test_transformer.py

```diff
@@ -0,0 +1,43 @@
+from pathlib import Path
+
+import pytest
+
+from robotidy.exceptions import InvalidParameterValueError
+
+from .. import TransformerAcceptanceTest
+
+
+class TestGenerateDocumentation(TransformerAcceptanceTest):
+    TRANSFORMER_NAME = "GenerateDocumentation"
+
+    def test_transformer(self):
+        self.compare(source="test.robot")
+
+    def test_transformer_overwrite(self):
+        self.compare(source="test.robot", expected="overwrite.robot", config=":overwrite=True")
+
+    def test_template_with_defaults(self):
+        template_path = Path(__file__).parent / "source" / "template_with_defaults.txt"
+        args = ["--transform", f"{self.TRANSFORMER_NAME};doc_template={template_path}"]
```

Comment:
```suggestion
        args = ["--transform", f"{self.TRANSFORMER_NAME}:doc_template={template_path}"]
```

Reply:
That makes me think that there should be one more test to check non-default template also.

Reply:
I have checked - ``;`` is required separator if value contains ``:``. But I will add additional test with relative path and ``:``. I will update the documentation with information on ``;` separator.